src/WebJobs.Script.WebHost/Resources/Functions/WarmUp/run.csx

```diff
@@ -1,13 +1,9 @@
-﻿using System.Net;
+﻿using Microsoft.AspNetCore.Mvc;
 
-public static Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
+public static IActionResult Run(HttpRequest req, TraceWriter log)
```

Comment:
Are these unrelated changes?

Reply:
We wanted these so the warmup/placeholder actually loads some more relevant dotnet core assemblies from disk and ngen to help with cold startup.

Reply:
This is just update to V2 new code for warm up function.